modin/experimental/xgboost/xgboost_ray.py

```diff
@@ -348,18 +321,26 @@ def create_actors(num_actors):
 
     Returns
     -------
-    tuple
-        Pair of actors list and placement group of actors.
+    list
+        List of pairs (ip, actor).
     """
     num_cpus_per_actor = _get_cpus_per_actor(num_actors)
-    pg = _create_placement_group(num_cpus_per_actor, num_actors)
+    node_ips = [
+        key for key in ray.cluster_resources().keys() if key.startswith("node:")
+    ]
+    num_actors_per_node = num_actors // len(node_ips)
+    actors_ips = [ip for ip in node_ips for _ in range(num_actors_per_node)]
+
     actors = [
-        ModinXGBoostActor.options(
-            num_cpus=num_cpus_per_actor, placement_group=pg
-        ).remote(i, nthread=num_cpus_per_actor)
-        for i in range(num_actors)
+        (
+            node_ip.split("node:")[-1],
+            ModinXGBoostActor.options(num_cpus=0.01, resources={node_ip: 0.01}).remote(
```

Comment:
Can we add `@ray.remote(num_cpus=0)` to the class? I think we might get rid of killing ray actors in that case.

Reply:
`@ray.remote(num_cpus=0)` was added to the class. In that case we can get rid of killing actors and waitings. Local tests are passed, let's wait CI tests.

Reply:
`@ray.remote(num_cpus=0)` excludes hangs, tests are passed. I propose to use that.

Reply:
Great!